docs/LowerBound_Dist_Profile_Derivation.ipynb

```diff
@@ -0,0 +1,2319 @@
+{
```

Comment:
<p>This is great!</p><p> </p><p>Eventually, we will need to convert all of these equations to base-zero indexing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
If I may ask, why do you think these equations are not base-zero indexing? You meant something like numpy indexing, right? 

I think it can be base-zero indexing. `i` is the start index of `T_i,m`. So, if someone has base-zero indexing in mind and set `i=0`, then the elements we are including in the `\sum \limits_{t=1}^{m} T[i+t-1]` will be from `0` to `m-1`.

Reply:
I just checked out the Welford Notebook...Got your point! Cool!

Reply:
<p>Great! It just makes it easier for people to convert it into code if they need it. In our case, it's probably not an issue because we are only going to use the final LB equations, which are pretty straightforward unlike the matrix profile or Welford equations (for MPX).</p>